### PR TITLE
Add a "Remove all space roles" button

### PR DESCRIPTION
**Pull request by rememberlenny**
remove

### CODE REVIEW

File: static_src/constants.js [hunk 1]
```diff
@@ -270,6 +270,10 @@ const userActionTypes = keymirror({
   USER_INVITE_ERROR: null,
   // Action to dismiss an user list notification.
   USER_LIST_NOTICE_DISMISSED: null,
+  // Action to remove all roles for space user.
+  USER_REMOVE_ALL_SPACE_ROLES: null,
+  // Action to removed all roles for space user.
```
Comment (jcscottiii) [inline static_src/constants.js]: `Action when all roles removed from space user`

File: static_src/stores/user_store.js [hunk 2]
```diff
@@ -164,6 +164,13 @@ export class UserStore extends BaseStore {
         break;
       }
 
+      case userActionTypes.USER_REMOVED_ALL_SPACE_ROLES: {
```
Comment (jcscottiii) [inline static_src/stores/user_store.js]: only need one more test. that it changes the list of users upon removing the person.

Comment (jcscottiii) [inline static_src/stores/user_store.js]: there's similar tests for the store for the add or delete cases.

File: static_src/constants.js [hunk 3]
```diff
@@ -270,6 +270,10 @@ const userActionTypes = keymirror({
   USER_INVITE_ERROR: null,
   // Action to dismiss an user list notification.
   USER_LIST_NOTICE_DISMISSED: null,
+  // Action when all roles for space user are removed.
```
Comment (jcscottiii) [inline static_src/constants.js]: oops i think i told you to put that comment on the wrong constant. i meant the one below should be the `when all`...

Comment (jcscottiii) [inline static_src/constants.js]: this first one should be `Action to...`

Comment (rememberlenny) [inline static_src/constants.js]: Fixed!

**Pull request merged**